packages/eslint-config-airbnb-base/rules/es6.js

```diff
@@ -156,6 +156,9 @@ module.exports = {
       'groups': ['builtin', 'external', 'internal', 'parent', 'sibling', 'index'],
       'newlines-between': 'never',
     }],
+    // Require modules with a single export to use a default export
+    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md
+    'import/prefer-default-export': 2,
```

Comment:
this makes this a breaking change - i'd prefer to cluster those all together. could we mark this with `// TODO: enable` and set it to `0` for now?